Headers/IOKit/80211/IO80211Controller.h

```diff
@@ -127,255 +127,197 @@ struct apple80211_lteCoex_report;
-        return false;
-    }
-    virtual bool setExtendedChipCounterStats(apple80211_stat_report *,void *) {
-        return false;
-    }
-    bool setChipCounterStats(apple80211_stat_report *,apple80211_chip_stats *,apple80211_channel *) {
-        return false;
-    }
-    virtual bool setLeakyAPStats(apple80211_leaky_ap_event *) {
-        return false;
-    }
-    bool setFrameStats(apple80211_stat_report *,apple80211_frame_counters *,apple80211_channel *) {
-        return false;
-    }
-    bool setPowerStats(apple80211_stat_report *,apple80211_power_debug_sub_info *) {
-        return false;
-    }
-     */
-
-	IOReturn copyIn(unsigned long long,void *,unsigned long);
-    void logIOCTL(apple80211req *);
-	bool isIOCTLLoggingRestricted(apple80211req *);
-
-	bool getBeaconPeriod(apple80211_beacon_period_data *);
-	SInt32 apple80211VirtualRequestIoctl(unsigned int,int,IO80211VirtualInterface *,void *);
-	bool getBSSIDData(OSObject *,apple80211_bssid_data *);
-	bool getSSIDData(apple80211_ssid_data *);
-	bool inputInfraPacket(mbuf_t);
-    void notifyHostapState(apple80211_hostap_state *);
-	bool isAwdlAssistedDiscoveryEnabled(void);
-    void joinDone(scanSource,joinStatus);
-    void joinStarted(scanSource,joinStatus);
-    void handleChannelSwitchAnnouncement(apple80211_channel_switch_announcement *);
-    void scanDone(scanSource,int);
-    void scanStarted(scanSource,apple80211_scan_data *);
-    void printChannels(void);
-    void updateInterfaceCoexRiskPct(unsigned long long);
-	SInt32 getInfraChannel(apple80211_channel_data *);
-    void calculateInterfacesAvaiability(void); // Suspected return type - int
-    void setChannelSequenceList(apple80211_awdl_sync_channel_sequence *); // Suspected return type - int
-    void setPrimaryInterfaceDatapathState(bool);
-	UInt32 getPrimaryInterfaceLinkState(void);
-    void setCurrentChannel(apple80211_channel *); // Suspected return type - int
-    void setHtCapability(ieee80211_ht_capability_ie *);
-    UInt32 getHtCapability(void);
-    UInt32 getHtCapabilityLength(void);
-	bool io80211isDebuggable(bool* enable);
-    void logDebug(unsigned long long,char const*,...); // Suspected return type - int
-    void vlogDebug(unsigned long long,char const*,va_list); // Suspected return type - char
-    void logDebug(char const*,...); // Suspected return type - int
-
-	bool calculateInterfacesCoex(void);
-	void setInfraChannel(apple80211_channel *);
-
-    void configureAntennae(void);
+    
+//public:
+//    IO80211SkywalkInterface* getInfraInterface(void);
+//    IO80211ScanManager* getPrimaryInterfaceScanManager(void);
+//    IO80211ControllerMonitor* getInterfaceMonitor(void);
+//    IOReturn addReporterLegend(IOService *,IOReporter *,char const*,char const*);
+//    IOReturn removeReporterFromLegend(IOService *,IOReporter *,char const*,char const*);
+//    IOReturn unlockIOReporterLegend(void);
+//    void lockIOReporterLegend(void);//怀疑对象，之前是返回int
```

Comment:
Why are these commented out? Also, can you replace Chinese with English here and below?

Reply:
non-virtual functions affects the link to the IO80211Family, and it is different from various versions of macOS, so I commented then now.
Yes, the comment I will change to English.

Reply:
Without calling these functions they will not link automatically. Could leave uncommented really.